<?xml version="1.0" encoding="UTF-8"?>
<class xmlns="http://xml.phpdox.net/src" full="PHPUnit_Framework_MockObject_BadMethodCallException" namespace="" name="PHPUnit_Framework_MockObject_BadMethodCallException" abstract="false" final="false" start="14" end="16">
  <file path="/home/robert/Desktop/Hogeschool/2ejaars Development/DevBOpdrachten/project56/http/laravel/vendor/phpunit/phpunit-mock-objects/src/Framework/MockObject/Exception" file="BadMethodCallException.php" realpath="/home/robert/Desktop/Hogeschool/2ejaars Development/DevBOpdrachten/project56/http/laravel/vendor/phpunit/phpunit-mock-objects/src/Framework/MockObject/Exception/BadMethodCallException.php" size="449" time="2015-12-17T18:44:10+00:00" unixtime="1450377850" sha1="794ebab235c491cd3dae1207b4c3c3d39a49b9b8" relative="vendor/phpunit/phpunit-mock-objects/src/Framework/MockObject/Exception/BadMethodCallException.php"/>
  <docblock>
    <description/>
    <since value="Class available since Release 2.0.6"/>
  </docblock>
  <extends full="BadMethodCallException" namespace="" name="BadMethodCallException"/>
  <implements full="PHPUnit_Framework_MockObject_Exception" namespace="" name="PHPUnit_Framework_MockObject_Exception"/>
</class>
